src/components/ZclAttributeReportingManager.vue

```diff
@@ -29,7 +29,10 @@ limitations under the License.
       data-cy="Attributes Reporting"
     >
       <template v-slot:body="props">
-        <q-tr :props="props">
+        <q-tr
+          :props="props"
+          v-show="props.row.side === 'server' || !enableServerOnly"
+        >
```

Comment:
Does matter have attribute reporting? Should this whole vue component not show up for matter?

Reply:
good point, ill add that